ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly-4.19-upgrade-from-stable-4.18.yaml

```diff
@@ -471,15 +471,15 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-marketplace
-- as: azure-ipi-proxy-workers-rhcos-rhel8-f28
-  cron: 48 13 16 * *
+- as: azure-ipi-proxy-f28
```

Comment:
Can we move it to `openshift-openshift-tests-private-release-4.19__multi-nightly-4.19-upgrade-from-stable-4.18.yaml` to use arm64 arch to run the update to align with e2e ci.